.github/workflows/issue.yml

```diff
@@ -17,10 +17,10 @@ jobs:
           body: |
             Thanks for the issue, we will take it into consideration! Our team of engineers is busy working on many types of features, please give us time to get back to you.
             
-            Feature requests that require more discussion may be closed. Read more about our [feature request process](https://forem.dev/foremteam/internal-rfc-process-and-forem-dev-discussions-3gl4) on forem.dev.
+            Feature requests that require more discussion may be closed. Read more about our [feature request process](https://forem.dev/foremteam/heads-up-github-discussions-and-feature-requests-54ff) on forem.dev.
```

Comment:
I went ahead and updated this link to reflect our latest feature request process!